Helper/Data.php

```diff
@@ -70,10 +70,15 @@ public function getActivityEndpoint()
 
     public function log($value)
     {
-        $writer = new \Zend\Log\Writer\Stream(BP . '/var/log/metrilo.log');
+        $logLocation = BP . '/var/log/metrilo.log';
+        if (file_exists($logLocation) && filesize($logLocation) > 104857600) {
```

Comment:
can you write this as `10 * 1024 * 1024` so that it is easier to understand later